shopping-cart/shopping-cart.js

```diff
@@ -0,0 +1,18 @@
+import cart from '../data/cart.js';
+import cameras from '../data/cameras.js';
+import { findById, calcOrderTotal, toUSD } from '../common/utils.js';
+import renderLineItem from '../shopping-cart/generate-line-items.js';
+
+const tbody = document.querySelector('tbody');
+const orderTotalCell = document.getElementById('subtotal-cell');
```

Comment:
This comes back as null because you have 'subtotal-cell' as a class. You either need to change it to an id in your html or use a query selector in this file